doctest/build.gradle

```diff
@@ -34,6 +34,22 @@ task bootstrap(type: Exec, dependsOn: ['cloneSqlCli']) {
 
 task startPrometheus(type: SpawnProcessTask) {
     doFirst {
+        def pidFile = new File(path, ".prom.pid.lock")
+        if (!pidFile.exists()) {
+            logger.quiet "No Prometheus server running!"
+        } else {
+            def pid = pidFile.text
+            def process = "kill $pid".execute()
+
+            try {
+                process.waitFor()
+            } finally {
+                pidFile.delete()
+                file("$projectDir/bin/prometheus").deleteDir()
+                file("$projectDir/bin/prometheus.tar.gz").delete()
```

Comment:
Does this cause Prometheus to be downloaded again immediately after?

Reply:
If it exists and would have been deleted had the previous run been successful, it will delete it then download it again after. Do you suggest if it exists we just go with the previously running version? I worried that if there were changes made to it, if it was not deleted it could create issues in testing.